plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/storage/ScaleIOStoragePool.java

```diff
@@ -214,6 +215,11 @@ public Map<String, String> getDetails() {
         return this.details;
     }
 
+    @Override
+    public String getLocalPathFor(String relativePath) {
```

Comment:
add défaut implementation as advised here: https://github.com/apache/cloudstack/pull/9270/files#r1719305288, and remove this here & other storage pool classes with same impl)